app/main.py

```diff
@@ -1 +1,56 @@
-# write your code here
+class SoftwareEngineer:
+    def __init__(self, name: str) -> None:
+        self.name = name
+        self.skills = []
+
+    def learn_skill(self, skill: str) -> None:
+        self.skills.append(skill)
+
+
+class FrontendDeveloper(SoftwareEngineer):
+    frontend_skills_list = ["JavaScript", "HTML", "CSS"]
+
+    def __init__(self, name: str) -> None:
+        super().__init__(name)
+        self.skills = FrontendDeveloper.frontend_skills_list
```

Comment:
when a class  inherits from SoftwareEngineer it automatically inherits an empty skills list. You should extend it instead of overriding it. see checklist